src/main/java/me/ryanhamshire/GriefPrevention/EntityDamageHandler.java

```diff
@@ -848,8 +848,8 @@ else if (damageSource instanceof Projectile arrow && arrow.getShooter() instance
     @EventHandler(ignoreCancelled = true, priority = EventPriority.LOWEST)
     public void onVehicleDamage(@NotNull VehicleDamageEvent event)
     {
-        //all of this is anti theft code
-        if (!instance.config_claims_preventTheft) return;
+        // all of this is vehicle anti theft code
+        if (!instance.config_claims_preventTheft || !instance.config_claims_protectVehicles) return;
```

Comment:
Yeah that's a good point. Since the conversations on the Discord indicate there'll be a full config rework soon, we may want to take that into consideration in a rewrite.

That said, I think there's something to be said for not letting the perfect get in the way of the good. I've been running this fork on my server for the past 10 hours or so and it has solved my particular use case and people have stopped complaining about their "minecarts that got stuck in someone else's claim."

I'd love for this to be merged back in so that others in my scenario can benefit. This is a fairly minimal change in-line with other config flags that requires minimal maintenance and moves the needle the right direction. I am also in no rush though as I can maintain the forked version since it is such a minimal divergence, at least until the new config update to this plugin drops.

Reply:
This is a good point. It is the lesser case, as this PR does maintain current defaults and is more preventative in nature. However, for existing servers that may have theft enabled, the best workaround for this is to use the existing value of `preventTheft` as the default for this setting.